packages/core/src/components/TextField/TextField.stories.mdx

```diff
@@ -74,6 +76,14 @@ To add the masking feature you need to provide `mask` prop. For now this feature
     <TextField id="fusion" type="Name" variant="fusion" label="Address" defaultValue="Demo" errorText="Something wrong" />
 </Preview>
 
+##### Character Count
+
+To display character count, you must provide a `maxLength` prop.
```

Comment:
Can we add `characterCount` also over here. Also can we change the prop name to `withCharacterCount` ?